app/views/notification_mailer/digest.html.erb

```diff
@@ -0,0 +1,29 @@
+<h2 class='digest-header'>
+  <% if @type == :digest %>
+    Your Notifications From The Past 24h
+  <% elsif @type == :instant %>
+    Your Recent Notifications
+  <% end %>
+</h2>
+
+<% @notifications.each do |project, project_notifications| %>
+  <div class='project-wrapper'>
+    <div class='project-name-wrapper'>
+      <% if defined?(Dradis::Pro) %>
```

Comment:
This could be removed if CE's `Project#name` returned something? Even an empty string

Reply:
Right now it returns "Dradis CE" by default. Which we opted not to have as the project title. Happy to return it in for parity though. The amount of people using mail with CE as mentioned is likely low.

This discussion can always come back up later.